Assets/Scripts/LaunchManager.cs

```diff
@@ -75,19 +79,37 @@ public class LaunchManager : MonoBehaviourPunCallbacks {
         return Random.Range(1000, 9999);
     }
 
+    public void DisplayMainMenu() {
+        __ResetPasscodePanel();
+        EnterEventPanel.SetActive(true);
+        PasscodeInputField.Select();
+        PasscodeInputField.text = string.Empty;
+        PhotonNetwork.Disconnect();

```

Comment:
Do we need to disconnect from lobby or can we just set panels active or inactive?

Reply:
Scratch that, we would need to change the current workflow if we don't disconnect here.